blame.go

```diff
@@ -13,17 +13,21 @@ import (
 	"srcd.works/go-git.v4/utils/diff"
 )
 
+// BlameResult represents the result of a Blame operation
 type BlameResult struct {
-	Path  string
-	Rev   plumbing.Hash
+	// Path is the path of the Blob object that we're blaming
+	Path string
+	// Rev is the hash of the specified Commit used to generate this result
+	Rev plumbing.Hash
+	// Lines contains the last person that modifies this line, and the content itself
```

Comment:
Not very clear. Lines contains lines, not person. Maybe reword:
`Lines contains the last person that modifies this line, and the content itself` -> `Lines contrains every line with its authorship`